docsrc/Makefile

```diff
@@ -22,7 +22,6 @@ github:
 	@make html
 	@rm -rf ../docs/*
 	@cp -r _build/html/* ../docs
-	@rm -rf ../docs/_modules
 	@rm -rf ../docs/_sources
```

Comment:
```suggestion

```

My bad, by "leave it" I meant we should delete this command and leave the sources folder alone